tests/foreman/api/test_product.py

```diff
@@ -35,407 +35,410 @@
 from robottelo.constants.repos import FAKE_1_PUPPET_REPO
 from robottelo.constants.repos import FAKE_1_YUM_REPO
 from robottelo.datafactory import invalid_values_list
+from robottelo.datafactory import parametrized
 from robottelo.datafactory import valid_data_list
 from robottelo.helpers import read_data_file
-from robottelo.test import APITestCase
 
 
-class ProductTestCase(APITestCase):
-    """Tests for ``katello/api/v2/products``."""
+@pytest.mark.tier1
+@pytest.mark.parametrize('name', **parametrized(valid_data_list()))
+def test_positive_create_with_name(request, name, module_org):
+    """Create a product providing different valid names
 
-    @classmethod
-    def setUpClass(cls):
-        """Set up organization for tests."""
-        super().setUpClass()
-        cls.org = entities.Organization().create()
+    :id: 3d873b73-6919-4fda-84df-0e26bdf0c1dc
```

Comment:
You'll need to include a testimony token here, because the test is parametrized.

`:parametrized: yes`

I'm going to try to add this to testimony validation so its part of CI checks (or pre-commit).

Reply:
Done.